src/resmom/mom_main.c

```diff
@@ -3730,6 +3734,46 @@ set_int(const char *id, char *value, int *var)
 	return HANDLER_SUCCESS;
 }
 
+/**
+ * @brief
+ *	set float value
+ *
+ * @param[in] id - function name
+ * @param[in] value - value
+ * @param[out] var - output float value
+ *
+ * @return      handler_ret_t
+ * @retval      HANDLER_FAIL            Failure
+ * @retval      HANDLER_SUCCESS         Success
+ *
+ */
+handler_ret_t
+set_float(const char *id, char *value, float *var)
+{
+	char	*left;
+	float	val;
+
+	if (value == NULL || *value == '\0') {
+		sprintf(log_buffer, "No value specified, no action taken.");
+		log_event(PBSEVENT_SYSTEM, PBS_EVENTCLASS_SERVER, LOG_NOTICE,
+			id, log_buffer);
+		return HANDLER_FAIL;	/* error */
+	}
+
+	val = strtod(value, &left);
+	if (left == value || val <= 0) {
+		sprintf(log_buffer, "bad value \"%s\"", value);
+		log_event(PBSEVENT_SYSTEM, 0, LOG_ERR, id, log_buffer);
```

Comment:
I'd use log_err() instead of log_event() here.

Reply:
I actually retained the implementation from set_int and other setter functions.